samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -57,10 +57,12 @@ static struct device *led_dev;
 static u32_t led_state;
 
 #if defined(CONFIG_NET_IPV6)
-static struct net_app_ctx udp6;
+static struct net_app_ctx app_udp6;
```

Comment:
Hrm.  At the end of the patch series, I end up with 2 lwm2m_ctx structures (each with it's own net_app_ctx).  They handle IPv4 and IPv6 individually.  Do we have any samples which illustrate dual use of a net_app_ctx?